tests/test_intero.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+'''Unit tests for the intero.py module.'''
+
+import pytest
+
+
+@pytest.fixture
+def mock_vim(mocker):
+    mock = mocker.MagicMock()
+    mocker.patch.dict('sys.modules', vim=mock)
+    return mock
+
+
+def test_stack_dirname(mock_vim):
+    from intero import stack_dirname
+
+    example_intero_stack_yaml = '/home/foobar/stack.yaml'
+    mock_vim.configure_mock(**{'eval.return_value': example_intero_stack_yaml})
+    assert stack_dirname() == '/home/foobar'
+
+
+def test_strip_internal(mock_vim):
+    from intero import strip_internal
+
+    example_prompt = '\ESC[38;2;255;100;0mλ> \ESC[m'
+    assert strip_internal(example_prompt) == example_prompt
```

Comment:
Ah, I was wondering what should happen there. Seems to have smoked a bug out of that function, will fix.